.github/workflows/gh-linux_mac.yml

```diff
@@ -0,0 +1,360 @@
+name: builds
```

Comment:
Shouldn't this file be removed? All of its individual jobs are now their own workflows, correct?